code/style.css

```diff
@@ -1,4 +1,401 @@
+  }
+
+  .desktop .travel:hover {
+    -webkit-text-decoration-color: rgb(0, 128, 0, 0.50);
+    text-decoration-color: rgb(0, 128, 0, 0.50);
+  }
+
+  .desktop .sports:hover {
+    -webkit-text-decoration-color: rgb(0, 0, 255, 0.50);
+    text-decoration-color: rgb(0, 0, 255, 0.50);
+  }
+
+  .desktop .society:hover {
+    -webkit-text-decoration-color: rgb(75, 0, 130, 0.50);
+    text-decoration-color: rgb(75, 0, 130, 0.50);
+  }
+
+  .desktop .weather:hover {
+    -webkit-text-decoration-color: rgb(238, 130, 238, 0.50);
+    text-decoration-color: rgb(238, 130, 238, 0.50);
+  }
+}
+/* END Header with logo, links and date only visible on desktop (min 1025px) */
+
+/* START Header with ticker always visible */
+@-webkit-keyframes ticker {
+  0% {
+    transform: translate3d(0, 0, 0);
+    visibility: visible;
+  }
+
+  100% {
+    transform: translate3d(-200%, 0, 0);
+  }
+}
+
+@keyframes ticker {
+  0% {
+    transform: translate3d(0, 0, 0);
+    visibility: visible;
+  }
+
+  100% {
+    transform: translate3d(-200%, 0, 0);
+  }
+}
+
+.ticker {
+  padding-right: 100%;
+}
+
+.ticker-wrap {
+  overflow: hidden;
+  height: 3rem;
+  background: linear-gradient(to right, rgba(28, 0, 0, 1), rgba(231, 46, 52, 1));
+  padding: 0.3rem 0 0 100%;
+}
+
+.ticker-wrap .ticker {
+  display: inline-block;
+  line-height: 1rem;
+  white-space: nowrap;
+  box-sizing: content-box;
+  -webkit-animation-iteration-count: infinite;
+  animation-iteration-count: infinite;
+  -webkit-animation-timing-function: linear;
+  animation-timing-function: linear;
+  -webkit-animation-name: ticker;
+  animation-name: ticker;
+  -webkit-animation-duration: 10s;
+  animation-duration: 10s;
+}
+
+.ticker-item {
+  display: inline-block;
+  padding: 0 1rem 0.3rem 1rem;
+  font-size: 1rem;
+}
+
+.ticker-item img {
+  width: 2rem;
+}
+/* END Header with ticker always visible */
+
+main {
+  padding: 0 0.5rem;
+}
+
+@media (min-width:1025px) {
+  main {
+    padding: 0 6rem;
+  }
+}
+
+/* START Feature section */
+.feature {
+  margin: 1rem 0;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-column-gap: 1.5rem;
+  padding: 1rem 1rem;
+}
+
+.feature-picture img {
+  width: 100%;
+}
+
+.feature-text {
+  color: white;
+}
+
+.feature-headline {
+  font-size: 1.5rem;
+  text-align: center;
+}
+
+.feature-lead {
+  display: none;
+}
+
+/* Opacity effect with white background so the text doesn't appear pink all over the feature section when hovering */
+@media (min-width: 1025px) {
+  .feature:hover {
+    background-color: white;
+    opacity: 0.8;
+  }
+
+  .feature {
+    grid-template-columns: 3fr 2fr;
+    align-items: center;
+  }
+
+  .feature-headline {
+    font-size: 1.75rem;
+    text-align: center;
+  }
+
+  /* Values to transform the overflow in an ellipsis, inspired by CodePen https://codepen.io/martinwolf/pen/qlFdp */
+  /* Can't manage to find why the ellipsis is displaying over the last letters of text instead of after in Sarafi */
```

Comment:
Very good that you have commented on things that don´t work in the code, so you can backtrack and fix the problem when you have read up upon Safari-specific display of ellipsis.